include/vkgcDefs.h

```diff
@@ -753,6 +757,7 @@ struct GraphicsPipelineBuildInfo {
   PipelineOptions options;  ///< Per pipeline tuning/debugging options
   bool unlinked;            ///< True to build an "unlinked" half-pipeline ELF
   bool dynamicVertexStride; ///< Dynamic Vertex input Stride is enabled.
+  bool enableUberFetchShader; ///< Use uber fetch shader
```

Comment:
It is better we can explain what is a Uber fetch shader.

Reply:
no problem, I will update the change description in next reversion.

Reply:
Can you explain Uber shader in this comments. I see you make descriptions in the commit comments. It is better to move them here.